LocalPubSubSpec: must publish to all local subscriber actors of a topic

https://github.com/akka/akka/runs/3239867889?check_suite_focus=true

```
must publish to all local subscriber actors of a topic *** FAILED *** (3 seconds, 117 milliseconds)
[info]   java.lang.AssertionError: timeout (3 seconds) during expectMessage while waiting for banana
[info]   at akka.actor.testkit.typed.internal.TestProbeImpl.assertFail(TestProbeImpl.scala:399)
[info]   at akka.actor.testkit.typed.internal.TestProbeImpl.expectMessage_internal(TestProbeImpl.scala:171)
[info]   at akka.actor.testkit.typed.internal.TestProbeImpl.expectMessage(TestProbeImpl.scala:149)
[info]   at akka.actor.typed.pubsub.LocalPubSubSpec.f$proxy1$2(LocalPubSubSpec.scala:43)
[info]   at akka.actor.typed.pubsub.LocalPubSubSpec.$init$$$anonfun$2$$anonfun$1(LocalPubSubSpec.scala:20)
```

--- a/akka-actor-typed-tests/src/test/scala/akka/actor/typed/pubsub/LocalPubSubSpec.scala
+++ b/akka-actor-typed-tests/src/test/scala/akka/actor/typed/pubsub/LocalPubSubSpec.scala
@@ -36,7 +36,9 @@ class LocalPubSubSpec extends ScalaTestWithActorTestKit with AnyWordSpecLike wit
         val statsProbe = testKit.createTestProbe[TopicImpl.TopicStats]()
         statsProbe.awaitAssert {
           fruitTopic ! TopicImpl.GetTopicStats(statsProbe.ref)
-          statsProbe.receiveMessage().localSubscriberCount should ===(3)
+          val stats = statsProbe.receiveMessage()
+          stats.localSubscriberCount should ===(3)
+          stats.topicInstanceCount should ===(1)
         }
 
         fruitTopic ! Topic.Publish("banana")

